src/plugins/intel_gpu/src/plugin/ops/split.cpp

```diff
@@ -34,6 +35,9 @@ static void CreateCommonSplitOp(ProgramBuilder& p, const std::shared_ptr<ov::Nod
     };
 
     auto inputs = p.GetInputInfo(op);
+    auto const_axis = std::dynamic_pointer_cast<ov::op::v0::Constant>(op->get_input_node_shared_ptr(1));
+    OPENVINO_ASSERT(const_axis != nullptr, "[GPU] Unsupported axis node type in ", op->get_friendly_name(), " (", op->get_type_name(), ")");
+    auto axis = const_axis->cast_vector<int64_t>()[0];
```

Comment:
What if axis is not const ? Shouldnt we handle that case too?

Reply:
Also I'd like to suggest to set the axis as normalized axis (i.e., if the rank is 4 and original axis is -1, set primitive axis as 3. And then, if the axis is -1, it is non-constant, (i.e., should be read from prev node), and if it has a valid number, it is constant and can be used as is.

Reply:
The original intention was to limit axis input to constant. But as your comment, it has been updated to also consider non-constant case and apply axis normalization.

Reply:
No I meant that, at first I thought we need to set some info whether the axis is constant or not to the primitive (i.e., by the value), so that we could decide whether to use mem input or axis attribute.
However now I see that you did the decision by "reordered input" ..  (so that the axis input will be always at the last input) 
It might cause (i.e., input order is different from primitive and ov::op) some confusion in the furture, 
Could you please add some information in the primitives/crop.hpp mentioning that the order of input is 
input, split length, and axis (optional) ?

Reply:
As your comment, I restored code to maintain the original input order. However separate information is not needed to check whether the axis is a constant input. and since axis is normalized as original comment, it seems sufficient to check whether the axis value is constant or not depending on whether it is negative or not.